app/models/role.rb

```diff
@@ -284,13 +284,6 @@ def self.permissions_with_descriptions
           'Client Access',
         ],
       },
-      can_search_own_clients: {
```

Comment:
Is this supposed to be in here?

Reply:
Yes!  Thanks, I think that's why the tests have been failing.